src/fee_lab_to_nwb/scherrer_ophys/scherrerophysimagingextractorinterface.py

```diff
@@ -1,39 +1,14 @@
-from typing import Optional
-
 from nwb_conversion_tools.datainterfaces.ophys.baseimagingextractorinterface import (
     BaseImagingExtractorInterface,
 )
-from nwb_conversion_tools.tools.roiextractors import write_imaging
-from nwb_conversion_tools.utils import OptionalFilePathType
-from pynwb import NWBFile
-
-from fee_lab_to_nwb.scherrer_ophys.scherrerophysimagingextractor import (
-    ScherrerOphysImagingExtractor,
-)
+from roiextractors.multiimagingextractor import MultiImagingExtractor
 
 
 class ScherrerOphysImagingExtractorInterface(BaseImagingExtractorInterface):
-    IX = ScherrerOphysImagingExtractor
+    IX = MultiImagingExtractor
 
-    def __init__(self, file_path: str):
-        super().__init__(file_path=file_path)
-        self.imaging_extractor = self.IX(file_path=file_path)
+    def __init__(self, imaging_extractors: list, timestamps: list):
+        super().__init__(imaging_extractors=imaging_extractors)
+        self.imaging_extractor = self.IX(imaging_extractors=imaging_extractors)
+        self.imaging_extractor.set_times(times=timestamps)
```

Comment:
I would only set the timestamps if they aren't regular - i.e., nest this in an `if check_regular(timestamps)` using whatever logical util you feel like for checking regularity.